portal/migrations/versions/38cfbd643c73_.py

```diff
@@ -0,0 +1,88 @@
+"""empty message
+
+Revision ID: 38cfbd643c73
+Revises: 5d7a1030065e
+Create Date: 2017-06-07 15:43:36.972833
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '38cfbd643c73'
+down_revision = '5d7a1030065e'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from portal.models.audit import Audit
+from portal.models.role import Role
+from portal.models.user import User, UserRoles
+
+
+Session = sessionmaker()
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = Session(bind=bind)
+
+    # get existing Observation.audit_id values
+    audit_ids = {}
+    for obs_id, audit_id in session.execute('SELECT id, audit_id FROM observations'):
+        audit_ids[obs_id] = audit_id
+
+    op.drop_constraint(u'observations_audit_id_fkey', 'observations', type_='foreignkey')
+    op.drop_column('observations', 'audit_id')
+    op.add_column('user_observations', sa.Column('audit_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(u'user_observations_audit_id_fkey', 'user_observations',
+                          'audit', ['audit_id'], ['id'])
+
+    # store existing audit_id values in UserObservations, based on observation_id
```

Comment:
@pbugni ah, fair point. How bout I just update that audit's user_id/subject_id with the UserObservation's user_id? Might be better than creating a bunch of new audits.